src/root.scss

```diff
@@ -1 +1 @@
-@import "../scss/core/root";
+@import "./core/root";
```

Comment:
Should we just move the `/src/core/_root.scss` file to `/src` and remove the underscore?